src/Translator/assets/dist/translator_controller.js

```diff
@@ -0,0 +1,229 @@
+import { IntlMessageFormat } from 'intl-messageformat';
+
+function formatIntl(id, parameters = {}, domain, locale) {
+    if (id === '') {
+        return '';
+    }
+    const intlMessage = new IntlMessageFormat(id, [locale.replace('_', '-')], undefined, { ignoreTag: true });
+    Object.entries(parameters).forEach(([key, value]) => {
+        if (key.includes('%') || key.includes('{')) {
+            delete parameters[key];
+            parameters[key.replace(/[%{} ]/g, '').trim()] = value;
+        }
+    });
+    return intlMessage.format(parameters);
+}
```

Comment:
1. Fixed, thanks
2. As said on Discord (:eyes:), the logic is taken from the Symfony Translator in its PHP version (https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/Translation/Formatter/IntlFormatter.php#L45), so I prefer keep the same logic instead.
3. Usually you want to quickly check if one of the concerned character can be found before using the regex, to prevent performances loss if you directly use the regex.